specs/multispec.yml

```diff
@@ -48,15 +48,16 @@ specs:
       prod: "rhel8"
       logos: "redhat-logos-httpd"
       preinstall_cmd: ['yum -y module enable {{ spec.module_stream }} httpd:2.4']
-      python_pkgs: ['{{ spec.pkg_prefix }}-setuptools', '{{ spec.pkg_prefix }}-pip',
-                    '{{ spec.pkg_prefix }}-virtualenv']
+      python_pkgs: ['{{ spec.pkg_prefix }}-setuptools', '{{ spec.pkg_prefix }}-pip']
       base_pkgs: ['nss_wrapper', 'httpd', 'httpd-devel', 'mod_ssl',
                   'mod_auth_gssapi', 'mod_ldap', 'mod_session',
                   'atlas-devel', 'gcc-gfortran', 'libffi-devel', 'libtool-ltdl enchant']
       extra_pkgs:
-        "2.7": ['{{ spec.pkg_prefix }}', '{{ spec.pkg_prefix }}-devel', 'glibc-langpack-en']
-        "3.6": ['python36', 'python36-devel']
-      ubi_versions: ['2.7', '3.6']
+        "2.7": ['{{ spec.pkg_prefix }}', '{{ spec.pkg_prefix }}-devel', 'glibc-langpack-en',
+                    '{{ spec.pkg_prefix }}-virtualenv']
+        "3.6": ['python36', 'python36-devel', '{{ spec.pkg_prefix }}-virtualenv']
```

Comment:
I'd use `{{ spec.pkg_prefix }}` everywhere, which means:
* remove `pkg_prefix` from Python 3.7 — it should not be used there
* redefine `pkg_prefix: "python3"` → `pkg_prefix: "python36"` for Python 3.6
* use `'{{ spec.pkg_prefix }}` for all `extra_pkgs` in RHEL 8.

WDYT?

Reply:
You're right this is a bit inconsistent, the problem is that we can't do this:

> * redefine `pkg_prefix: "python3"` → `pkg_prefix: "python36"` for Python 3.6

Because in RHEL8 the python3 packages are called "python3-foo", but the python3 interpreter is called "python36" - the prefix is different for each.

However, I could use `{{ spec.pkg_prefix }}` for `python38{,-devel}` so that it's unified with python2. Does that sound ok?

Reply:
Yes, please do.

Reply:
Amended the commit, please take a look!